noisemaker/src/components/Piano.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import MIDISounds from "midi-sounds-react";
+// import Key2 from "./Key2";
+import "./Piano.css";
+
+class Piano extends Component {
+  playKey1() {
```

Comment:
Cool! I wonder if there is a way to make this more DRY by passing some value(s) in through the event into just a single handler.

Reply:
For example, the button with `playKey1` might have a value of 2, which you pass into your `playChordNow()` function as the first argument.

Reply:
Yeah, some sort of mapping function to store data and make it easier to modify later too.